tools/update_chip_support.py

```diff
@@ -39,7 +39,6 @@
     "stm32f412g": ["stm32f4xx"],
     "stm32f429zi": ["stm32f4xx"],
     "stm32f446re": ["stm32f4xx"],
-    "swervolf-eh1": ["swerv"],
```

Comment:
I suspect we should add `veer_el2`

Reply:
From what I see, this dict is described as
```
# Static info of chip crates that just support other chips.
```
And previously the chip was implemented in two crates: `swervolf-eh1` and `swerv`, so the entry was present.

Right now the implementation uses only the `veer_el2` chip. I  don't think any change would be needed here, please let me know if it's otherwise.